superset-websocket/tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
+      "outDir": "dist",
+      "target": "es6",
```

```suggestion
      "target": "es2019",
```

Node 12 supports es2019 syntax